addons/account/models/account.py

```diff
@@ -233,7 +233,7 @@ def _default_outbound_payment_methods(self):
     sequence_id = fields.Many2one('ir.sequence', string='Entry Sequence',
         help="This field contains the information related to the numbering of the journal entries of this journal.", required=True, copy=False)
     refund_sequence_id = fields.Many2one('ir.sequence', string='Refund Entry Sequence',
-        help="This field contains the information related to the numbering of the refund entries of this journal.", copy=False)
+        help="This field contains the information related to the numbering of the refund entries of this journal.", copy=False, ondelete="restrict")
```

Comment:
I wouldn't put ondelete=restrict on the refund_sequence_id field because:
* there is no such restriction on sequence_id field
* ir.sequence is an advanced notion, normally when you mess with that you're supposed to know what to do